eth/watchers/types.go

```diff
@@ -13,3 +14,7 @@ type BlockWatcher interface {
 type EventWatcher interface {
 	Subscribe(sink chan<- types.Log) event.Subscription
 }
+
+type OrchestratorStore interface {
```

Comment:
Doesn't seem like we need to export this interface - can we make it internal to the package?

Reply:
fixed in 6a397072